tests/integration/annotation_import/conftest.py

```diff
@@ -493,29 +502,37 @@ def configured_project(client, ontology, rand_gen, image_url):
         data_row_ids=data_row_ids,
         wait_processing_max_seconds=DATA_ROW_PROCESSING_WAIT_TIMEOUT_SECONDS,
         sleep_interval=DATA_ROW_PROCESSING_WAIT_SLEEP_INTERNAL_SECONDS)
```

Comment:
the below `create_batch` function calls `_wait_until_data_rows_are_processed` internally, so no need for this here